test/unit/engine/NavigationProperty.js

```diff
@@ -35,8 +35,7 @@ describe("NavigationProperty", function () {
         },
         entityType: {
           getLegacyApiObject: () => {},
-          key: [
```

Comment:
Could you run `npx gulp prettier` in the project before commit. It looks like you have use different version of prettier than is in the project now.